### DISCUSSION TITLE
Metadata metadata

### DISCUSSION THREAD

**Discussion by savage13** [usgs/gems-tools-pro#52] [General]
Putting this here as the metadata documentation (or links) in gems is somewhat lacking.

How the FGDC metadata is produced
https://desktop.arcgis.com/en/arcmap/latest/tools/conversion-toolbox/export-metadata.htm
It uses the translator ARCGIS2FGDC.xml

More info on editing the Metadata in the GDB files 
https://desktop.arcgis.com/en/arcmap/latest/manage-data/metadata/creating-and-managing-fgdc-metadata.htm

Details on the FGDC format and the fields are at:
https://desktop.arcgis.com/en/arcmap/latest/manage-data/metadata/illustrated-guide-to-complete-fgdc-metadata.htm

I also found this to be helpful to understand the xml format and required fields:
https://www.fgdc.gov/csdgmgraphical/index.html

You can also find the metadata-parser at:
https://geology.usgs.gov/tools/metadata/tools/doc/mp.html

It is the same tool that gets used by https://www1.usgs.gov/mp/ and in gems to validate the metadata.

**Comment by aldunn4** [comment]
I am doing an overhaul on the metadata for our geodatabases right now, and I appreciate any tips and tricks anyone has to share.

**Reply by tfelger** [reply]
I am successfully using the Jupyter notebook samples that are provided with the USGS Metadata Wizard 2.0.7 to streamline the process of batch updating multiple metadata xmls. If you are not already a user of Metadata Wizard, you can find more information here: https://code.usgs.gov/usgs/fort-pymdwizard.

**Reply by aldunn4** [reply]
I did download that and didn't get very far in playing around with it. I think I am at the point of starting with a metadata-free geodatabase and copying and pasting metadata elements from our existing geodatabase, editing as needed in Arc. I know nothing about Jupyter notebooks and once we get the metadata stable it won't need much updating. 

If I remember correctly, it was mentioned at DMT 2023 that the Metadata Wizard is no longer being actively worked on at the USGS.

**Reply by ethoms-usgs** [reply]
Sorry, that was me. No, it's definitely still being worked on. What I meant to say was that after I had suggested some additional features to the Metadata Wizard project lead, the response was that it was unlikely that any major new changes would be made because of a focus in the group to transition to ISO. I am sure they will continue to address smaller issues and I have heard nothing about it being discontinued.

**Reply by aldunn4** [reply]
Thanks for clarifying.

**Reply by ethoms-usgs** [reply]
If you find MW cumbersome, you might take a look at [Tkme](https://geology.usgs.gov/tools/metadata/tools/doc/tkme.html). It's fast, simple, and, I think, more intuitively shows you the layout of a metadata record than MW with its tabs and nested dialogs.
